mnefun/_status.py

```diff
@@ -102,11 +105,11 @@ def print_proc_status(p, subjects, structurals, analyses, run_indices):
             these_steps.append('missing')
         steps_all.append(these_steps)
 
-    steps = ['raw', 'sco', 'pbd', 'coreg',
```

Comment:
yeah I know, but the column widths are determined by the shortest column, so 3 char is the limit